### DISCUSSION TITLE
Under which scope does `Create Variable` block creates variables?

### DISCUSSION THREAD

**Discussion by jucamo8311** [postmanlabs/postman-flows#6] [Q&A] [answered]
Hi

Flows looks like a very interesting functionality.

I have a question , its not important but a notice that when I execute flow "Start button" , prints double entry on the console for each service called, is this intentional ?

Another question that I have is , the functionality "create variables "under witch scope will create those variables (Global,Collection,Environment,Data,Local)?

Amazing work by the way!

**Comment by saswatds** [comment]
> I have a question , its not important but a notice that when I execute flow "Start button" , prints double entry on the console for each service called, is this intentional ?

No this is not intentional, could you refresh the page once to see if it's resolved, we will try to find the root cause.

**Comment by saswatds** [comment, marked as answer]
> Another question that I have is , the functionality "create variables "under witch scope will create those variables (Global,Collection,Environment,Data,Local)?

The `Create Variable / Assign Variable` blocks both create variable in `Data` scope
